rust/private/rustc.bzl

```diff
@@ -198,7 +198,8 @@ def rustc_compile_action(
         transitive = [
             toolchain.rustc_lib.files,
             toolchain.rust_lib.files,
-        ],
+            # TODO wait for CcToolchainInfo.all_files to be available
+        ] + ctx.files._cc_toolchain,
```

Comment:
Is ctx.files._cc_toolchain a depset of files? Should it just go into the transitive list?

Reply:
Tried that, Linux (at least) complains. I am borrowing a Mac to test against tonight.